src/attribution/onINP.ts

```diff
@@ -233,6 +234,78 @@ const getIntersectingLoAFs = (
   return intersectingLoAFs;
 };
 
+const getLoAFSummary = (attribution: INPAttribution) => {
+  const loafAttribution: LongAnimationFrameSummary = {};
+
+  // Stats across all LoAF entries and scripts.
+  const interactionTime = attribution.interactionTime;
+  const inputDelay = attribution.inputDelay;
+  const processingDuration = attribution.processingDuration;
+  let totalStyleAndLayout = 0;
+  let totalForcedStyleAndLayout = 0;
+  let totalScriptTime = 0;
+  let numScripts = 0;
+  let slowestScriptDuration = 0;
+  let slowestScript: PerformanceScriptTiming | null = null;
+  let slowestScriptPhase: string = '';
+  const phases: Record<string, Record<string, number>> = {} as Record<
+    string,
+    Record<string, number>
+  >;
+
+  attribution.longAnimationFrameEntries.forEach((loafEntry) => {
+    totalStyleAndLayout +=
+      loafEntry.startTime + loafEntry.duration - loafEntry.styleAndLayoutStart;
+    loafEntry.scripts.forEach((script) => {
+      const scriptEndTime = script.startTime + script.duration;
+      if (scriptEndTime < interactionTime) {
+        return;
+      }
+      totalScriptTime += script.duration;
+      numScripts++;
+      totalForcedStyleAndLayout += script.forcedStyleAndLayoutDuration;
+      const blockingDuration =
+        scriptEndTime - Math.max(interactionTime, script.startTime);
+      const invokerType = script.invokerType; //.replace('-', '_');
+      let phase = 'processingDuration';
+      if (script.startTime < interactionTime + inputDelay) {
+        phase = 'inputDelay';
+      } else if (
+        script.startTime >
+        interactionTime + inputDelay + processingDuration
+      ) {
+        phase = 'presentation';
+      }
+      if (!(phase in phases)) {
+        phases[phase] = {};
+      }
+      if (!(invokerType in phases[phase])) {
+        phases[phase][invokerType] = 0;
+      }
```

Comment:
I think this is Baseline Widely available, right?
```suggestion
      phases[phase] ??= {};
      phases[phase][invokerType] ??= 0;
```

Reply:
Looks to be! Updated.